src/inmanta/data/__init__.py

```diff
@@ -306,7 +308,7 @@ def get_accessor(self, column_name: str, table_prefix: Optional[str] = None) ->
         """
         return the sql statement to get this column, as used in filter and other statements
         """
-        table_prefix_value = "" if table_prefix is None else table_prefix + "."
+        table_prefix_value = self.table_prefix_dot if table_prefix is None else table_prefix + "."
```

Comment:
Why make a separate variable with the dot if it doesn't simplify its usage? This could be simply
```suggestion
        table_prefix_value = (self.table_prefix if table_prefix is None else table_prefix) + "."
```

Reply:
Not a change request per se.

Reply:
Need to clean this up

Reply:
There are two different overrides here, that is why this odd construction exists. 

(this whole thing needs refactoring)

Reply:
Still not a change request per se, but just FYI, nothing seems to be changed here. Did you forget?